frontend/src/auth/components/LoginPage/LoginPage.tsx

```diff
@@ -35,11 +35,17 @@ export class LoginPage extends React.PureComponent<Props, State> {
     super(props);
 
     this.state = {
-      username: state.username,
+      username: '',
       password: ''
     };
   }
 
+  private onKeyPress = (event: any) => {
```

Comment:
@giovannicimolin, weird that this code is required when in plain old HTML it works automatically :man_shrugging:

Reply:
@lgp171188 I think that this is happening because the fields are not in a form.